view/frontend/web/js/view/payment/omise-offsite-placeorder.js

```diff
@@ -12,7 +12,6 @@ define(
         'use strict';
```

Comment:
Just my opinion, 
Understand that the reason of renaming this, is because this file contains only `placeOrder` function. 
However, I think `placeorder` here is a bit improper name for its purpose. 

The `omise-offsite-placeorder.js` sounds like it is a function for placing an order that can be used individually while in fact this file is an abstract file that is extending from the `base-method-renderer` and will (have to) be extended by an actual payment method's renderer file(s).

Reply:
so what do you suggest to be done @guzzilar ?

Reply:
if I may, I would like to suggest to leave it as it is : )

Reply:
what do you mean saying:  _in fact this file is an abstract file that is extending from the base-method-renderer_
this is file that contains method that is used in other files.
this is not really abstract, it has just defined function/functionality that is used in other files.

in my opinion word `base` is not proper to be left here.

Reply:
i like `omise-offsite-placeorder.js`

Reply:
It is not an abstract by its own code, but the concept of itself is to be extended, right? 
i.e. if we look at the omise-offsite-alipay-method.js
```js
define(
    [
        'Omise_Payment/js/view/payment/omise-offsite-placeorder',
    ],
    function (
        Base,
    ) {
        return Component.extend(Base).extend({
            // ...
        });
    }
);
```

In a sense of `omise-base-method-renderer.js` > `omise-offsite-method-renderer.js` > `omise-offsite-alipay-method.js`

I would support the opinion of changing the filename if in fact it is in another way around.
```js
define(
    [
        'Omise_Payment/js/view/payment/omise-base-method-renderer',
        'Omise_Payment/js/view/payment/omise-offsite-placeorder',
    ],
    function (
        Base,
        Placeorder,
    ) {
        return Object.assign({}, Base, Placeorder, {
            // ...
        });

    }
);
```

Reply:
it is not actually to be extended, we do not take X and add X+a
we really only use X
before I just simply used this, but we changed to be extended.
`Omise_Payment/js/view/payment/omise-offsite-method-renderer` it is not good in my opinion, we do not render anything here.